SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -2331,6 +2331,8 @@ void CSolver::AdaptCFLNumber(CGeometry **geometry,
                              CSolver   ***solver_container,
                              CConfig   *config) {
 
+  /// TODO: Add OpenMP stuff to this method.
+
   /* Adapt the CFL number on all multigrid levels using an
    exponential progression with under-relaxation approach. */
```

Comment:
This is the only outstanding large loop used in a fluid iteration, but I could use some clarification. It seems that the final values of max/min/ave CFL end up being based only on the last visited grid (there is an outer loop over mesh levels).
Is this by design or were these values intended to be min/max/avg over all levels?

Reply:
@economon, you seem to be around more often these days, can you have a look at this question please.

Reply:
Yep, you are right.. this was just an oversight. Since the min/max/avg are just for reporting, it is probably best to report only the fine grid in the output. We can just add if (iMesh == MESH_0) { ... } around the relevant parts.